scripts/install_gradio.sh

```diff
@@ -0,0 +1 @@
+pip install -e .
```

Comment:
This will only work if this bash script is called from the root gradio folder. Might want to put a comment on top clarifying that.